types/packages/caver-rpc/src/klay.d.ts

```diff
@@ -0,0 +1,217 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { AccountKey } from '../../caver-account/src'
+import {
+    Block,
+    BlockNumber,
+    TransactionReceipt,
+    BlockWithConsensusInfo,
+    TransactionForRPC,
+    PromiEvent,
+    TransactionForSendRPC,
+    RLPEncodedTransaction,
+    DecodedAnchoringTransaction,
+    Log,
+    LogsOptions,
+    LogObject,
+    CallObject,
+} from '../../caver-core/src'
+import { Transaction } from '../../caver-transaction/src'
+import RpcCallToMethod from '../../caver-rtm/src'
+
+export interface AccountForRPC {
+    accType: number
+    account: {
+        nonce: number
+        balance: string
+        humanReadable: boolean
+        key: AccountKeyForRPC
+    }
+}
+
+export interface AccountKeyForRPC {
+    keyType: number
+    nonce?: number
+    key?: AccountKeyForRPC | AccountKeyForRPC[]
+}
+
+export interface Syncing {
+    startingBlock: number
+    currentBlock: number
+    highestBlock: number
+    knownStates: number
+    pulledStates: number
+}
+
+export interface KlaytnCall {
```

Comment:
Why this is defined as separate interface?

Reply:
This will be used in Abstractransaction and Validator